Resources/Prototypes/Catalog/VendingMachines/Inventories/dinnerware.yml

```diff
@@ -26,3 +26,4 @@
     DrinkMugOne: 1
     DrinkMugRainbow: 2
     DrinkMugRed: 2
+    FoodPlateMuffinTin: 5
```

Comment:
Put it below the FoodPlateTin. The objects show up in the machine in the order they are defined here, so they should be ordered by category.